lib/App/Sqitch.pm

```diff
@@ -45,7 +45,8 @@ has verbosity => (
     is       => 'ro',
     lazy     => 1,
     default  => sub {
-        shift->config->get( key => 'core.verbosity' ) // 1;
+        my $self = shift;
+        $self->options->{verbosity} // $self->config->get( key => 'core.verbosity' ) // 1;
```

Comment:
Huh. Odd. I wonder how I missed this. Needs a test.

Reply:
I basically just caught it because I was trying to get some actual debug output when I ran into trouble earlier this week.. ;)

Reply:
@theory, I added tests for verbosity handling in `t/base.t`:

https://github.com/theory/sqitch/pull/362/commits/994730025e867721a066ab85f5eb31cfd879fe75

Is this a good place for it?